css/selectors/past.json

```
@@ -8,7 +8,7 @@
           "spec_url": "https://drafts.csswg.org/selectors/#the-past-pseudo",
           "support": {
             "chrome": {
-              "version_added": false
+              "version_added": "≤83"
```

WebVTT came with Chrome 23 so I would set it to 23.

```suggestion
              "version_added": "23"
```